### ISSUE TITLE
Has anyone been able to run it with Cassandra? Or at least Oauth2_Provider?

### ISSUE THREAD

**Issue by Gittinhub** [pennersr/django-allauth#3119]
Hi there!

So, did anyone a workaround it to use in Cassandra?

At least Oauth2_Provider since it does the hard part of authenticating every request?

It is just useful for Sql databases?

**Comment by amanrawat121** [comment]
@Gittinhub you got any solution???

**Comment by pennersr** [comment]
Closing -- this does not point to an issue in allauth.

**Comment by Gittinhub** [comment]
@amanrawat121  No, I did not but I expect to get it someday. I Asked the same at django_cassandra_engine and a person involved (r4fek/django-cassandra-engine)said he is on it.
See hee: https://github.com/r4fek/django-cassandra-engine/pull/161#issuecomment-1247841430

I see now you commented there too LOL... 
So, that's where I am too, waiting something from @r4fek ... Let me know if you have any news too.

**Comment by pennersr** [comment]
https://codeberg.org/allauth/django-allauth/issues/3119
